File: raspberrypi/led.c
/*************************************************************************
	> File Name: led.c
	> Author: life
	> Created Time: Sun 22 Mar 2020 15:03:49 GMT
 ************************************************************************/

#include<stdio.h>
#include <wiringPi.h>

int main(void) {

   wiringPiSetup();

   pinMode (13, OUTPUT);

   for(;;) {

       digitalWrite(13, HIGH);delay (500);

       digitalWrite(13, LOW);delay (500) ;

    }

}

File: raspberrypi/led.py
import RPi.GPIO as GPIO
import time

GPIO.setmode(GPIO.BCM) 
GPIO.setup(27,GPIO.OUT)
GPIO.setwarnings(False)

try:
    while True:
        GPIO.output(27,GPIO.HIGH)
        time.sleep(1)
        GPIO.output(27,GPIO.LOW)
        time.sleep(1)
except:

     print("except")
